lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,368 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
```

Comment:
ahh -- i didn't realize that there were well-known false positives. so keep it `false`, but make sure to prominently feature it in the docs for folks who aren't regularly bitten by it?